src/model/model.ts

```diff
@@ -1228,7 +1229,16 @@ function getDefaultHistoryQueryOpts(): HistoryQueryOpts {
 class InputModel {
     historyShow: OV<boolean> = mobx.observable.box(false);
     infoShow: OV<boolean> = mobx.observable.box(false);
+    aIChatShow: OV<boolean> = mobx.observable.box(false);
     cmdInputHeight: OV<number> = mobx.observable.box(0);
+    aiChatTextAreaRef: React.RefObject<HTMLTextAreaElement>;
+    aiChatWindowRef: React.RefObject<HTMLDivElement>;
+    codeSelectBlockRefArray: Array<React.RefObject<HTMLElement>>;
```

Comment:
I wonder how readable this is 
code select is the name for storing the code blocks to select and copy them, and all the chat related variables are stored as ai chat or ai assistant chat, or cmd info chat. Maybe I should call them all ai cmd info for future clarity